vendor-extra/JatsContentBundle/src/ViewConverter/FrontSubjectGroupContentHeaderVisitor.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\ViewConverter;
+
+use DOMNodeList;
+use FluentDOM\DOM\Document;
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\ConvertsChildren;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+use function implode;
+
+final class FrontSubjectGroupContentHeaderVisitor implements ViewConverterVisitor
+{
+    use ConvertsChildren;
+    use SimplifiedVisitor;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function doVisit(Element $object, View $view, array &$context = []) : View
+    {
+        /** @var Document $document */
+        $document = $object->ownerDocument;
+        $xpath = $document->xpath();
+        $xpath->registerNamespace('jats', 'http://jats.nlm.nih.gov');
+
+        // @todo - add support for other ways of expressing a subject group in JATS.
```

Comment:
This all comes down to the 'which JATS' question. For now we'll keep just this way (since you can always write code  yourself to add others).